### ISSUE TITLE
dials.reciprocal_lattice_viewer loses some integrated reflections if you prod it

### ISSUE THREAD

**Issue by benjaminhwilliams** [dials/dials#688]
Steps to reproduce:

1. Open an `integrated_experiments.json` and `integrated.pickle` with `dials.reciprocal_lattice_viewer`.
2. (Make the spots bigger so you can see them.)
3. Click on 'unindexed'.  Unsurprisingly, you only see the unindexed spots now.
4. Click on 'all'.  Most of the integrated spots don't return.  It seems perhaps that only the indexed strong spots return.

I can provide a good real-world example from a recent data collection on I19.

Questions:
* Am I being daft?
* Is this known?

**Comment by benjaminhwilliams** [comment]
@ndevenish, is this a duplicate of #671, or something new?

**Comment by ndevenish** [comment]
Hmm, possibly. Previously, I think you would never see unindexed on the integrated.pickle at all?

Can have look myself but in theory only once I’m back.

**Comment by dagewa** [comment]
I could not confirm this with a randomly-chosen data set

**Comment by benjaminhwilliams** [comment]
Agreed, I can no-longer reproduce.  I suspect some other RLV changes have squashed this.  Perhaps @ndevenish has some insight.

**Comment by ndevenish** [comment]
There was a really bad bug in the reciprocal atmosphere that we fixed a couple of months ago, perhaps that was the same issue

**Comment by jbeilstenedmands** [comment]
Unclear if still an issue, closing
